_data/whats-new.yml

```diff
@@ -4,8 +4,71 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Mon Aug 26 22:12:19 2019
+updated: Mon Sep 9 16:43:06 2019
 entries:
+- description: Added an example of the `escapeCss` function to the [Templates XSS
+    security](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/templates/template-security.html)
+    topic.
+  versions: 2.3.x
+  type: Major update
+  date: September 6, 2019
+  link: https://github.com/magento/devdocs/pull/5299
+- description: Added more information about the `output` attribute for containers
+    in the [Layout Instructions topic](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-instructions.html).
+  versions: 2.2.x, 2.3.x
+  type: Major update
+  date: September 6, 2019
+  link: https://github.com/magento/devdocs/pull/5332
+- description: 'Updated the Cloud Release notes for [ece-tools version 2002.0.21](https://devdocs.magento.com/guides/v2.3/cloud/release-notes/cloud-tools.html#v2002021):<br/>-
+    Added environment variables for Cloud deployment: [CONSUMERS_WAIT_FOR_MAX_MESSAGES](https://devdocs.magento.com/guides/v2.3/cloud/env/variables-deploy.html#consumers_wait_for_max_messages)
```

Comment:
```suggestion
    Added environment variables for Cloud deployment: [`CONSUMERS_WAIT_FOR_MAX_MESSAGES`](https://devdocs.magento.com/guides/v2.3/cloud/env/variables-deploy.html#consumers_wait_for_max_messages)
```

Reply:
Thanks @shrielenee !  I updated that in the PR description and pushed the change.